src/canvas-extractor/canvas_extractor/api/canvas_helper.py

```diff
@@ -0,0 +1,86 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
```

Comment:
I prefer having a "helpers" directory for functions that are not calling the API.

Reply:
The to_df() should only be called by code in this directory.  The remove_duplicates() is more generic, but right now only code in this directory uses it.  We could split it off, but I'd wait to move it up a level until something outside this directory wants to use it.